x-pack/package.json

```diff
@@ -95,7 +95,6 @@
     "@types/jsdom": "^16.2.3",
     "@types/json-stable-stringify": "^1.0.32",
     "@types/jsonwebtoken": "^7.2.8",
-    "@types/lodash": "^4.14.159",
```

Comment:
got `@types/lodash/ts3.1 error TS2300: Duplicate identifier` error since both oss & x-pack types were included. I'd remove `lodash` from `x-pack/package.json` completely to rely on `oss/package.json`, but some plugins use `import/no-extraneous-dependencies` @elastic/kibana-operations